src/components/welcome-screen/welcome-screen.e2e.test.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Enzyme, {shallow} from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import WelcomeScreen from "./welcome-screen";
+
+Enzyme.configure({
+  adapter: new Adapter()
+});
+
+describe(`WelcomeScreenComponent`, () => {
+  it(`WelcomeScreenButton should be pressed`, () => {
```

Comment:
хорошо, если название теста отражает его суть и хорошо читается по английски, например, можно так:  
```suggestion
  it(`should run callback when clicking WelcomeScreenButton`, () => {
```